### ISSUE TITLE
Attendee - Ticket Wallet - Ticket Wallet/Tickets Index

### ISSUE THREAD

**Issue by Mcscott142** [revelrylabs/bn-mobile-react#42]
## Background 
The ticket wallet will have all of the ticket histories for a user. 

Latest Mocks are kept in the sketch file in this [folder](https://www.dropbox.com/sh/gt7c74ndhk7me5w/AADIxnNPe39Xbq3HJUCXfb9xa?dl=0). Please doublecheck that this screen has not changed before working ticket. Latest screens are in the production ready tab. 

## Scenario: A user selects the Ticket Wallet icon on the nav
- [ ] When I am directed to the ticket wallet page
- [ ] Then I see 3 tabs that contain tickets:
	- [ ] Upcoming
	- [ ] Past
	- [ ] Transferred

## Mocks
![ios_app_-_dev_version_sketch](https://user-images.githubusercontent.com/6877299/43805326-f24409fe-9a64-11e8-9ed6-623fb31d7a11.png)

![ios_app_-_dev_version_sketch](https://user-images.githubusercontent.com/6877299/43805337-fb1a0088-9a64-11e8-8d00-525d249c9ff1.png)

**Comment by brittanygay** [comment]
Gonna go ahead and pick this up to start working on once #54 is done.

**Comment by zekewell** [comment]
This passes for

- Upcoming Events 
- Past events

Does not pass for transferred events, but based on the mocks it looks like that got left on on purpose.

**Comment by iamjoshfrank** [comment]
Agree with Max - the mocks don't include a Transfer tab, so I'm going to move Transfer to a new issue so that it can be clarified.

Passes otherwise.
